pkg/config/config.go

```diff
@@ -806,9 +806,35 @@ func IsValidDeviceMode(mode string) bool {
 	return true
 }
 
+// resolveHomeDir converts a path referencing the home directory via "~"
+// to an absolute path
+func resolveHomeDir(path string) (string, error) {
+	// check if the path references the home dir to avoid work
+	// don't use strings.HasPrefix(path, "~") as this doesn't match "~" alone
+	// use strings.HasPrefix(...) to not match "something/~/something"
+	if !(path == "~" || strings.HasPrefix(path, "~/")) {
+		// path does not reference home dir -> Nothing to do
+		return path, nil
+	}
+	
+	// only get HomeDir when necessary
+	home, err := unshare.HomeDir()
+	if err != nil {
+		return "", err
+	}
+
+	// replace the first "~" (start of path) with the HomeDir to resolve "~"
+	return strings.Replace(path, "~", home, 1), nil
+}
+
 // isDirectory tests whether the given path exists and is a directory. It
 // follows symlinks.
 func isDirectory(path string) error {
+	path, err := resolveHomeDir(path)
+	if err != nil {
+		return err
+	}
+
```

Comment:
The code is fine, my only hesitancy is the function name.  When I first read it, I thought you were somehow resolving to the home directory.  However after 10 minutes of pondering, I don't have a much better name other than "handleTilde" or perhaps "resolveTilde".  Thoughts?

Reply:
Thanks for the feedback.
At first I had the code embedded in isDirectory, but extracted it for seperation and readability purposes.
The nameing wasn't perfect for me too, but it was the most descriptive since it resolves (the tilde to) the home directory path. 

Some ideas:
- resolveTildeToHomeDir
- pathReplaceTilde
- resolveTilde (as you already said)

Reply:
@snapstromegon, I do very much like that you've extracted the code to a separate function. 
Your names got me thinking.  "replaceTildeInPath" maybe?